tests/person/test_models.py

```diff
@@ -45,10 +30,10 @@ def test_get_person(self):
         """
         Test retrieving a person
         """
-        person = self._create_person('Test_Person_0')
+        person = Person.create(external_id='Test_Person_0')
         kf_id = person.kf_id
 
-        person = self._get_person(kf_id)
+        person = Person.find_one(kf_id=kf_id)
```

Comment:
`get()` would be preferable for performance and readability, but there is an issue where we have multiple primary keys here.

See [get()](http://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.get) for why it would be preferable.